mods/email/web/test.css

```diff
@@ -0,0 +1,3 @@
+.foo {
+color: aliceblue;
+}
```

Comment:
Do you want ot push this?

Reply:
wait, that wasn't mine... I'll delete the file